src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+const employees = document.querySelectorAll('li');
+
+function sortList(someEmployees) {
+  const sortedEmployees = [...someEmployees].sort((item1, item2) => {
+    const salary1 = +item1.dataset.salary
+      .split(',').join('')
+      .split('$').join('');
+
+    const salary2 = +item2.dataset.salary
```

Comment:
Use Number for [type casting](https://github.com/mate-academy/style-guides/blob/master/javascript.md#223-numbers-usenumberfor-type-casting-andparseintalways-with-a-radix-for-parsing-strings)
```suggestion
    const salary2 = Number(item2.dataset.salary)
```